music_server.py

```diff
@@ -7,90 +7,84 @@
 
 app = Flask(__name__)
 
+
 @app.route('/')
 def server_home_page():
-	return "This is the server's home page. Nothing interesting to see here."
+    return "This is the server's home page. Nothing interesting to see here."
+
 
 @app.errorhandler(Exception)
 def page_not_found(e):
-	print(e)
-	return '404 Not Found', 404
+    print(e)
+    return '404 Not Found', 404
+
 
 @app.route('/allalbums')
 def get_albums_with_tracks():
-	"""
-	Retrieve a complete list of albums and the contents
-	of each album on the server.
-	"""
+    """
+    Retrieve a complete list of albums and the contents
+    of each album on the server.
+    """
+    # Create an empty dictionary for all albums
+    all_albums = {}
 
-	# Initialize an empty dictionary to store albums and their tracks
-	albums={}
+    directory_name = 'Music'
 
-	# Relative path to the Music directory with Albums
-	directory_name = 'Music'
+    album_list = os.listdir(directory_name)
+    album_list.remove('.DS_Store')
```

Comment:
Can you please add a conditional check around this? Without the check, this causes issues on Windows since this folder does not exist.